src/packages/sdk/src/engineCommands.ts

```diff
@@ -218,12 +218,14 @@ export async function getConfig({
     }
   }
 
+  // engineArgs is temporary until we officially add the mongoDb preview flag.
+  const engineArgs = ['--enable-experimental=mongoDb']
```

Comment:
I'd like to only enable this if the provider is `mongodb`, but I think this needs to be set before we've parsed the data model. If you know of a better way, please let me know.

Reply:
Additionally, where should I write engine command tests?